adapters/AppLovin/AppLovinAdapter/GADMAdapterAppLovinUtils.h

```diff
@@ -34,6 +34,16 @@ NS_ASSUME_NONNULL_BEGIN
 + (ALIncentivizedInterstitialAd *)incentivizedInterstitialAdWithZoneIdentifier:
                                       (NSString *)zoneIdentifier sdk:(ALSdk *)sdk;
 
+/// Check whether or not the AppLovin native ad has all the required assets to map to a
+/// Unified native ad.
++ (BOOL)containsRequiredUnifiesNativeAssets:(ALNativeAd *)nativeAd;
+
+/// Check whether or not the AppLovin native ad has all the required assets to map to a
+/// App Install native ad.
++ (BOOL)containsRequiredAppInstallNativeAssets:(ALNativeAd *)nativeAd;
+
++ (void)log:(NSString *)format, ...;
```

Comment:
I understand the lack of abstraction for the `log` method bothers you, so feel free to abstract this method away from  other places it's being used